include/rocksdb/db.h

```diff
@@ -1046,6 +1046,19 @@ class DB {
     return IngestExternalFile(DefaultColumnFamily(), external_files, options);
   }
 
+  // CreateColumnFamilyWithImport() will create a new column family with
+  // column_family_name and import external SST files specified in metadata into
+  // this column family.
+  // (1) External SST files can be created using SstFileWriter.
+  // (2) External SST files can be exported from a particular column family in
+  //     an existing DB.
+  // External files are deleted on a successful return.
```

Comment:
Idea was to cleanup column family in case of any errors. Earlier change was only doing DropCF on failure. Added DestroyCF as well now and that should ensure that there should not be any CF leftover. Added test to cover this as well.

Documented the external set file handling on success/failure cases.